src/ripple/app/ledger/LedgerMaster.cpp

```diff
@@ -710,9 +710,18 @@ class LedgerMasterImp
 
         if (!ledger)
         {
-            // FIXME: We should really only fetch if the ledger
-            //has sufficient validations to accept it
+            if ((seq != 0) && (getValidLedgerIndex() == 0))
```

Comment:
I personally prefer to make the comparison explicit when it's an integer and zero has some special meaning.